airflow/contrib/hooks/telegram_hook.py

```diff
@@ -0,0 +1,88 @@
+import telegram
+from time import sleep
+from airflow.hooks.base_hook import BaseHook
+from airflow.exceptions import AirflowException
+
+
+def telegram_retry(func, *args, **kwargs):
```

Comment:
It will look better when it's a context manager. Check if the code already exists in the project.

Reply:
Or as a decorator of the whole method, not just a single call.